sdk/jobs/E2E-ds-experience/e2e-ml-workflow-part-I.ipynb

```diff
@@ -0,0 +1,1225 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "1dbce700",
+   "metadata": {},
+   "source": [
+    "# Using Azure ML Pipelines to Productionize E2E ML Workflows: Part I"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0bbe4d4d",
+   "metadata": {},
+   "source": [
+    "**Learning Objectives** - By the end of this two part tutorial, you should be able to use Azure Machine Learning (Azure ML) to productionize your ML project.\n",
+    "\n",
+    "This means you will be able to leverage the AzureML Python SDK to:\n",
+    "\n",
+    "- Connect to your Azure ML workspace\n",
+    "- Create Azure ML Datasets\n",
+    "- Create reusable Azure ML Components\n",
+    "- Create, validate and run Azure ML pipelines\n",
+    "- Deploy the newly-trained model as an endpoint\n",
+    "- Call the Azure ML endpoint for inferencing"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0113b218",
+   "metadata": {},
+   "source": [
+    "**Motivations** - This tutorial is intended to introduce Azure ML to data scientists who want to scale up or publish their ML projects. By completing a familiar end-to-end project, which starts by loading the data and ends by creating and calling an online inference endpoint, the user should become familiar with the core concepts of Azure ML and their most common usage. Each step of this tutorial can be modified or performed in other ways that might have security or scalability advantages. We will cover some of those in the Part II of this tutorial, however, we suggest the reader use the provide links in each section to learn more on each topic."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4c00c3c3",
+   "metadata": {},
+   "source": [
+    "**Requirements** - In order to benefit from this tutorial, you need to have:\n",
```

Comment:
Nit: the first list starts lines with capital letters, this list starts with lowercase. Either is fine, but you should be consistent.